dev/src/reference.ts

```diff
@@ -1977,26 +1950,89 @@ export class Query<T = DocumentData> {
             before: !this._queryOptions.startAt.before,
           })
         : undefined;
-    } else {
-      if (this._queryOptions.hasFieldOrders()) {
-        structuredQuery.orderBy = this._queryOptions.fieldOrders.map(o =>
-          o.toProto()
-        );
+    }
+
+    const reqOpts: api.IRunQueryRequest = {
+      parent: parentPath.formattedName,
+      structuredQuery,
+    };
+
+    reqOpts.transaction = transactionId;
```

Comment:
Nit: Merge this into the assignment above.

Reply:
Done.